tests/test_other.py

```diff
@@ -8241,6 +8241,17 @@ def test_separate_dwarf(self):
     self.assertNotIn(b'subdir/output.wasm.debug.wasm', wasm)
     self.assertNotIn(bytes(os.path.join('subdir', 'output.wasm.debug.wasm'), 'ascii'), wasm)
 
+    # Check that the dwarf file has only dwarf and name sections
+    debug_wasm = webassembly.Module('subdir/output.wasm.debug.wasm')
+    section_names = [sec.name for sec in debug_wasm.sections()]
+    if 'name' not in section_names:
+      self.fail('name section not found in separate dwarf file')
```

Comment:
We recently added `Module.has_name_section` which does the first part of this. This does need to check for a second custom section, so that's not enough. But maybe we should replace that function with a method to get the names, and use that here and there?

Reply:
I replaced lines 8244-8247 with the the straightforward `if not debug_wasm.has_name_section()`.
But I'm not sure what you are suggesting for the rest. I'm asserting about both the types and names of the other sections, so I think just iterating over the section descriptors rather than the names makes the most sense?